src/win/tty.c

```diff
@@ -1611,6 +1627,13 @@ static int uv_tty_write_bufs(uv_tty_t* handle,
     uv_buf_t buf = bufs[i];
     unsigned int j;
 
+  if (vterm == SUPPORTED) {
+    utf16_buf_used = MultiByteToWideChar(CP_UTF8, 0, buf.base, buf.len, NULL, 0);
+    MultiByteToWideChar(CP_UTF8, 0, buf.base, buf.len, utf16_buf, utf16_buf_used);
```

Comment:
We are converting to wide char (UTF16) regardless. In the current implementation (i think) we are taking whatever is passed in (UTF8) mapping it to what is best supported and converting it to wide char (UTF16). In this case we are simply converting straight to UTF16 without mapping anything since it should be all supported.